drivers/serial/ptmx.c

```diff
@@ -276,7 +277,7 @@ int ptmx_register(void)
 {
   /* Initialize driver state */
 
-  nxsem_init(&g_ptmx.px_exclsem, 0, 1);
+  nxmutex_init(&g_ptmx.px_lock);
```

Comment:
ditto. either keep like it is or add `nxmutex_destroy` if `register_driver` fails

Reply:
next patch